source/MaterialXTest/MaterialXRenderOsl/Utilities/raytype_background.osl

```diff
@@ -0,0 +1,9 @@
+surface raytype_background(color Cin = 1, string envmap_filename = "")
+{
+    color bgColor = Cin;
+    if (!raytype("camera"))
+    {
+        bgColor = environment(envmap_filename, I);
+    }
+    Ci = bgColor * emission();
```

Comment:
environment shaders should use the `background()` closure (no practical difference, just a convention)

Reply:
After more discussion over slack -- I missed this was assigned to a quad.

But a simpler solution is proposed here: https://github.com/AcademySoftwareFoundation/OpenShadingLanguage/pull/1649

that should avoid the need for creating an extra quad entirely.

Reply:
Agreed with @fpsunflower that removing the quad will provide a more robust solution, though we may need to take this in stages, as I encounter some stretching artifacts when viewing the environment through transmissive surfaces with that approach.